src/main/scala/diplomaticobjectmodel/DiplomaticObjectModel.scala

```diff
@@ -0,0 +1,25 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.diplomaticobjectmodel
+
+import freechips.rocketchip.diplomaticobjectmodel.model.OMComponent
+
+import scala.collection.mutable.ListBuffer
+
+class DomCollector {
```

Comment:
It's a little confusing that you use the acronym "DOM" in this file but "OM" everywhere else. Can you rename both the class and the instance variables to be consistent with the existing convention?

Reply:
Fixed